pacman/staticfiles/js/tableSort.js

```diff
@@ -0,0 +1,42 @@
+/*
+IF im going to be brutally honest, this vm runs at 13fps,
+and writing any code is a chore, especially with how often characters are not registered
+YES, this is stolen straight from W3Schools, and I do not care. . . 
```

Comment:
Cite it instead of a comment like this? Provide a link as a comment and remove this unnecessary comment. Something like
// Function from W3Schools
// LINK
Comments exist to help with the code - if you need to let me know something leave a github comment and not a code comment.

Reply:
oh yea tbh, completely forgot that i left that, was kinda just fuming after searching for solutions, not really trying to say anything, i guess leaving lore? lol will be fixed next commit